### ISSUE TITLE
Compressor makeup scaling makes it impossible to use

### ISSUE THREAD

**Issue by tavasti** [quamplex/geonkick#24]
When trying to use compressor, I find adjusting makeup nearly impossible. Usable values are so low, that with that minimal field it would be adjusting one pixel. With my eyesight and hand accuracy with mouse, impossible. When ever trying to use it, I get clipped / distorted audio.

**Comment by iurienistor** [comment]
yes, there i need work more on this to make compressor more usable and adjust parameters. lower the amplitude from general settings, and use makeup to encreas the resulted volume. sorry for this. the compressor is per total output,  i recomnend to use a dedicated compressor for now.

**Comment by tavasti** [comment]
Yeah, and global compressor might even get removed, because it can be replaced with external. If compressor and distortion would per layer, then they would be something which can't be replaced with external stuff.

**Comment by iurienistor** [comment]
There will be needed a input control for compressor and drive, thus, there will not be needed to lower the amplitude of the graph shown, but limit the input to effect. Actually every effect as a standard has an input and output limiter in order to avoid clipping if using in a chain of effects. I missed this.
